src/app/portlets/content-types/fields/content-type-field-dragabble-item/content-type-field-dragabble-item.component.scss

```diff
@@ -38,6 +38,7 @@ dot-icon {
     justify-content: center;
     flex-direction: column;
     flex-grow: 1;
+    overflow: hidden;
```

Comment:
This fix, prevent this to happen on long fields. 

![image](https://user-images.githubusercontent.com/31667212/51049685-33358780-1594-11e9-98c2-044df3a8c882.png)